codec-http/src/test/java/io/netty/handler/codec/http/multipart/HttpPostMultiPartRequestDecoderTest.java

```diff
@@ -358,4 +358,121 @@ private void commonNotBadReleaseBuffersDuringDecoding(HttpDataFactory factory, b
         factory.cleanAllHttpData();
         decoder.destroy();
     }
+
+    // Issue #11668
+    private void commonTestFileDelimiterLFLastChunk(HttpDataFactory factory, boolean inMemory)
```

Comment:
nit: static

Reply:
OK, I follow the same way than the other test in the very same Junit class. I will change this one to static.

Reply:
Done for both sub methods private (existing one and new one) ;-)